examples/planout-example/Readme.md

```diff
@@ -78,6 +78,7 @@ To run this example go to it's root directory and run `npm install`.
 
 You can then run:
 
+    $ npm run build
```

Comment:
Can you update this guy for V2? https://github.com/yahoo/mendel/blob/master/examples/Readme.mdown